src/WP_CLI/LanguagePackUpgrader.php

```diff
@@ -45,6 +45,7 @@ public function generic_strings() {
 	 *
 	 * @param string $package The URI of the package. If this is the full path to an
```

Comment:
You need to adapt the alignment between the two `@param` comments.

Reply:
fixed in https://github.com/wp-cli/language-command/pull/82/commits/9830213cd0d7d55bfe279e5d8b9082df85ed796e